code/modules/mob/living/carbon/human/species/species.dm

```diff
@@ -155,7 +159,13 @@
 
 	for(var/organ in has_organ)
 		var/organ_type = has_organ[organ]
+		//world << "[organ_type]"//debug print
 		H.internal_organs_by_name[organ] = new organ_type(H,1)
+		H.internal_organs += new organ_type
```

Comment:
In fact, you might want to make organ/internal/New() insert on the targeted mob, so you don't get any lists updated multiple times with the same organ, and don't have to fuss with that silly extra "internal" argument for regular organ New().